ui/src/overriden/frontsite/Home/Slogan.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Container, Grid, Header } from 'semantic-ui-react';
+
+export const Slogan = ({ ...props }) => {
+  return (
+    <Container className="container-header">
+      <Grid>
+        <Grid.Column width={16} textAlign="left">
+          <Header as="h1" className="fs-headline-header" size="huge">
+            CERN Library Catalogue
+          </Header>
+          <Header.Subheader className="fs-headline-subheader">
+            Find books fast and easily.
```

Comment:
It looks like a marketing tagline! :)
What about: `Books, e-books, papers, standards at CERN` or similar?